<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Koniec</class>
 <widget class="QWidget" name="Koniec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>End</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/witcher.png</normaloff>:/icons/witcher.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelWiadomosc">
       <property name="text">
        <string>Informacja</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelKoniec">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/cards.qrc">:/ikony/wygrana.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelG1">
           <property name="text">
            <string>Gracz 1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEditG1">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>25</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelG2">
           <property name="text">
            <string>Gracz 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEditG2">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>25</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/cards.qrc">
         <normaloff>:/ikony/tak.png</normaloff>:/ikony/tak.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/cards.qrc"/>
 </resources>
 <connections/>
</ui>
